src/components/flowchart-wrapper/flowchart-wrapper.scss

```
@@ -1,12 +1,30 @@
 @import '../../styles/variables';
 @import '../../styles/mixins';
 
+$total-sidebar-toolbar-width-closed: $sidebar-width-closed +
+  $global-toolbar-width;
+$total-sidebar-toolbar-width-open: $sidebar-width-open + $global-toolbar-width;
```

Nit: I think we can get away without `total-` here.

```suggestion
$sidebar-toolbar-width-closed: $sidebar-width-closed +
  $global-toolbar-width;
$sidebar-toolbar-width-open: $sidebar-width-open + $global-toolbar-width;
```